postgres/client/init.sls

```diff
@@ -7,7 +7,7 @@
   {%- endif %}
 {%- endfor %}
 
-{%- if postgres.use_upstream_repo == true %}
+{%- if 'pkg_repo' in postgres and postgres.use_upstream_repo == true %}
```

Comment:
Here, it is not strictly required, since underlying logic in `postgres.upstream` will handle it anyway.
But does no harm as well.

Reply:
Yes, I know. Just thought that being consistent in the checks within the state file would make it easier to follow the logic.

I can remove those checks if you prefer and just leave the strictly required ones.

Reply:
I mention this only beacase the `upstream.sls` has the notification state, which explicitly shows message about additional repository will not be configured.
This was done for some clarity, then the `postgres.use_upstream_repo` was set, but actually will not going to have any effect.

Reply:
So, would you like me to remove the check? :smile:

Reply:
I think it would be better, but up to you :smile:

Reply:
Done